test/locustJob.yaml

```diff
@@ -0,0 +1,53 @@
+apiVersion: batch/v1
+kind: Job
+metadata:
+  labels:
+    job-name: benchmark
+  name: benchmark
+  namespace: open-cluster-management
+spec:
+  backoffLimit: 6
+  completions: 1
+  parallelism: 1
+  template:
+    metadata:
+      creationTimestamp: null
+      labels:
+        job-name: benchmark
+    spec:
+      imagePullSecrets:
+      - name: search-pull-secret
+      containers:
+      - image: quay.io/stolostron/search-indexer:locust
+        imagePullPolicy: Always
+        name: benchmark
+        resources: {}
+        terminationMessagePath: /dev/termination-log
+        terminationMessagePolicy: File
+        env:
+        - name: N_CLUSTERS
+          value: "10"
+        - name: SPAWN_RATE
+          value: "1.0"
+        - name: DB_USER
+          valueFrom:
+            secretKeyRef:
+              key: database-user
+              name: search-postgres
+        - name: DB_PASS
+          valueFrom:
+            secretKeyRef:
+              key: database-password
+              name: search-postgres
+        - name: DB_NAME
+          valueFrom:
+            secretKeyRef:
+              key: database-name
+              name: search-postgres
+        - name: HOST
+          value: search-indexer-open-cluster-management.apps.sno-4xlarge-413-xxxxx.dev07.red-chesterfield.com
```

Comment:
You can access the service directly without a route or cluster specific name. I'll need to confirm the syntax, but it's something like this:
```suggestion
        - name: HOST
          value: search-indexer.open-cluster-management.svc
```